app/views/posts/index.html.erb

```diff
@@ -1 +1,41 @@
-<h1>Here are list of all posts for given user</h1>
\ No newline at end of file
+<div class="container border">
+  <div class="row">
+    <div class="col-md-4 p-3" >
+      <div class="img-cover text-center">
+        <img src="<%= @posts[0].author.photo%>" alt="not found" width="180">
+      </div>
+    </div>
+    <div class="col-md-8 p-3" >
+      <div class="border p-4"><h3><%= @posts[0].author.name%></h3>
+      <h6 class="pull-right text-right">Number of posts:<%= @posts[0].author.posts_counter%></h6></div>
+    </div>
+  </div>
+
+  <% @posts.each_with_index do |post,index| %>
```

Comment:
- [ ] Kindly make sure to show correctly the `posts#index` page, while running the page there's an error on it

![image](https://github.com/Elhambasir/blog-app/assets/93630700/57740a90-d3c5-4778-b48b-b0925f372051)